frontend/packages/client/src/components/ProposalCreate/StepTwo.js

```diff
@@ -47,7 +47,10 @@ const StepTwo = ({ stepData, setStepValid, onDataChange }) => {
 
     // push now if date is today and not already in time interval
     if (cutOffDate) {
-      const nowDate = new Date();
+      const nowDate =
+        process.env.NODE_ENV === 'production'
```

Comment:
Need to introduce a new environment variable like `APP_ENV` because staging uses `NODE_ENV=production`

https://github.com/DapperCollectives/CAST/blob/main/frontend/Dockerfile.stage#L11